src/LLVM_Runtime_Linker.cpp

```diff
@@ -1029,6 +1031,9 @@ std::unique_ptr<llvm::Module> get_initial_module_for_target(Target t, llvm::LLVM
             if (t.has_feature(Target::AVX2)) {
                 modules.push_back(get_initmod_x86_avx2_ll(c));
             }
+            if (t.has_feature(Target::AVX512_SapphireRapids)) {
+                modules.push_back(get_initmod_x86_avx512_ll(c));
```

Comment:
Should this LL module be named x86_avx512_sapphirerapids? I am wondering if people might add other things to avx512.ll that aren't intended to be gated on this flag.

Reply:
Alternatively: maybe we can just add this module if we have Target::AVX512, since usage of the function is already gated on this flag in CodeGen_X86.cpp?